fuzz/oss-fuzz-build.sh

```diff
@@ -21,7 +21,7 @@ cmake \
     -DBUILD_TESTING=ON \
     -DBUILD_SHARED_LIBS=OFF \
     -DBUILD_EXAMPLES=NO \
-    -DENABLE_SECURITY=NO \
+    -DENABLE_SECURITY=ON \
```

Comment:
I did run into some trouble, it appears it was caused by flipping this switch. I think the trouble is ultimately caused by erroneously trying to integrate wrapper libraries for the plugins where the plugins are not getting built. On consideration it is somewhat silly to waste all those CPU cyclones on building the tests when we don't even want to run those tests. The reason the tests need to be built is that we want to access symbols that are not ordinarily exported by the library, but it seems to me it would make more sense to support that directly with a build option (e.g., `EXPORT_ALL_SYMBOLS`).

#1974 should take care of that, if that gets approved, then the build should go faster and not exhibit this problem anymore.

Reply:
I also ran into trouble building this, but only after I cleared out all build artifacts. I merged master and it does build.